spaceapi_server/src/lib.rs

```diff
@@ -15,9 +16,10 @@ use std::sync::Mutex;
 use std::sync::Arc;
 
 use rustc_serialize::json::{Json, ToJson};
-use iron::{Request, Response, IronResult, Iron, Set};
+use iron::prelude::*;
```

Comment:
I think the iron::prelude is designed to be imported that way. Also it was less work for me :wink: